styles/components/_menu.scss

```diff
@@ -1,60 +1,69 @@
 .page__container {
-  width: 100vw;
-  height: calc(100vh - 2rem);
-  background-color: white;
-  text-align: center;
+  z-index: -1;
+  height: 100vh;
+  background: url(/static/images/ui/background.png) no-repeat center;
+  background-size: cover;
 }
 
 .menu-window {
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  padding: 60px 0 60px 60px;
   &__header {
-    position: relative;
-    display: flex;
-    flex-direction: column;
-    padding: 32px;
-  }
-  &__heading {
-  }
-  &__description {
+    font-family: Roboto;
+    font-weight: 700;
   }
 }
 
 .menu {
-  position: relative;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 32px;
-  &__graphic {
-    &-container {
-      width: 30vw;
-      height: 50vh;
-      background-color: cadetblue;
-    }
-  }
   &__buttons {
     display: flex;
+    width: 20%;
     flex-direction: column;
-  }
-  &__button {
-    &--rules {
+    margin-bottom: 55vh;
```

Comment:
I think this margin might be crashing the layout on my screen, can you try a different approach to the units?

Reply:
I'll check, but this project is for desktop size.